dmv/dmv.js

```diff
@@ -19,13 +19,18 @@ DMV.prototype.currentCustomerFor = function(agent) {
 }
 
 DMV.prototype.nextCustomer = function() {
+  if (this.customers.length === 0) {
+    throw new Error('No customers')
+  }
   const currentCustomer = this.customers.shift()
-  this.agentList.find(agent => {
+  let currentAgent = null;
+  this.agentList.forEach(agent => {
     if (!this.attending[agent]) {
       this.attending[agent] = currentCustomer
-      return true
+      currentAgent = agent
     }
   })
+  return {agent: currentAgent, customer: currentCustomer}
 }
 
```

Comment:
Is this method behavior well defined or documented?

Reply:
I'd like to see better documentation

Reply:
forEach is a better choice. `well defined` method behavior indeed.

Reply:
what do you think?

Reply:
why would you only use let when defining another function and for null?

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"